### ISSUE TITLE
Why does ActonDB create test DBs etc on startup?

### ISSUE THREAD

**Issue by plajjan** [actonlang/acton#585]
Here's a normal startup of ActonDB:
```
kll@Boxy:~/terastream/acton$ dist/bin/actondb
SERVER: Using args: portno=32000, gportno=34000, verbosity=1, seeds:
Test create ACTORS_TABLE - OK (0)
Test create MSGS_TABLE - OK (0)
Test create_state_schema - OK
BACKEND: Queue table 2 created
Test create MSG_QUEUE table - OK (0)
Test create_queue_schema - OK
SERVER: Started [0.0.0.0:32000, 127.0.0.1:34000], my_lc = VC(99536:0)
```

What are all these "Test create ..." messages about? That doesn't really seem like something we should normally run, right @aagapi?

**Comment by aagapi** [comment]
Those message are somewhat misleading, i.e. they shouldn't include the word 'test', but the code is currently needed. It creates the needed actors table and IPC queue table on server side for Acton on server startup. A more orthodox alternative to this would be to have this triggered by the RTSs at startup (including needed concurrency handling for multiple RTSs, as well as generic schema manipulation functions in the DB API), but since currently the RTS and DB are pretty tightly integrated, we currently do schema creation for Acton server side automatically upon DB server startup. I will remove the 'Test ' prefix from the messages.

**Comment by plajjan** [comment]
There are more of them than you fixed in the PR....

**Comment by plajjan** [comment]
I fixed the two other occurrences in #588.

**Comment by plajjan** [comment]
All occurrences fixed and merged. Closing.
